.circleci/config.yml

```diff
@@ -182,6 +185,35 @@ jobs:
           path: frontend/reports/
       - store_artifacts:
           path: frontend/coverage/
+  dynamic_security_scan:
+    executor: machine-executor
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: Start up local server
+          command: |
+            docker-compose -f docker-compose-test.yml run --rm server yarn install --production=false
```

Comment:
Why we can't use the `docker-compose` at the root of the project? I know I'm the one who suggested using a different compose file but I can't remember why

Reply:
My memory is that we couldn't use the original docker compose (that was made for local dev) because we need this to run the way it would really in production: a backend server that redirects requests to a FE static site.

Reply:
ooh gotcha I think that was the reason, thanks for reminding me!